portal/templates/assignments/teacher-assignments.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'layouts/base.html' %}
+
+{% block title %} Assignments{% endblock %}
+
+
+{% block content %}
+<form method='post'>
+  {% for student in students%}
+  <h2>{{ student['first_name']}}--{{student['last_name']}}</h2>
+  <h3>Assignment name</h3>
```

Comment:
This says the words "Assignment name" but it doesn't actually give an assignment name. Is it supposed to do that?

Reply:
And if student['name'] is the assignment name then having this is a little confusing and should have some comments or change the name from name to assignment_name.

Reply:
As well as both the h3 and h4 could be combined to look something like this: <h3>Assignment Name: {{ student['assignment_name'] }}</h3>